Update/vector symbols

LanesGood opened the pull request:
This PR:

- Adds different colors for vector layer points, and border
- Adds zoom-dependent radius
- Makes legend collapsible, sets legend max height

<img width="1440" height="788" alt="image" src="https://github.com/user-attachments/assets/6ec26fcc-969f-4424-b1c5-cb03be6bd331" />

Closes #102

LanesGood committed:
Assign multiple vector layer colors

LanesGood committed:
Add halo and zoom-based size change to vector layer circles

LanesGood committed:
Set max height on legend, make legend collapsible

willemarcel reviewed (approved):
It's working well. Some suggestions: green/mint/teal and blueLight/indigo are too similar, could we use tones that give more contrast?

LanesGood committed:
Update color palette

LanesGood commented:
Thanks @willemarcel. I've updated the color palette using colors from ColorBrewer. I also moved some of the Vector color order around so that ideally there will not be similar colors adjacent to eachother.
